docs/design/kademlia_audit_gating.md

```diff
@@ -0,0 +1,116 @@
+# Kademlia Audit Gating
+
+## Abstract
+
+StorageNode B is added to StorageNode A's routing table only if StorageNode A trusts Satellite C and Satellite C has verified StorageNode B is not disqualified and has a high enough audit count and uptime count. If StorageNode B does not have this verification, it is added to StorageNode A's Routing Table Antechamber.
```

Comment:
"... and has a high enough audit count and uptime count" is likely implicit in "is not disqualified".  I would make these details parenthetical.  Later versions of disqualification may include other criteria.

Reply:
got it!